src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -63,7 +64,24 @@ public JsonWebKeySet()
         {
         }
 
-#pragma warning disable CS0618 // Type or member is obsolete
+        /// <summary>
+        /// Initializes an ECDsa Adapter.
+        /// </summary>
+        /// <remarks>
+        /// As ECDsa Adapter is not supported on some platforms, PlatformNotSupported exception will be swallowed and logged.
+        /// </remarks>
+        static JsonWebKeySet()
+        {
+            try
+            {
+                ECDsaAdapter = new ECDsaAdapter();
+            }
+            catch (PlatformNotSupportedException ex)
```

Comment:
If we catch all exceptions here then we are better protected. Otherwise, even if the keys being translated never have a ECD key, we could bring as server down.

Reply:
The only exception that we could throw is _PlatformNotSupportedException_, but I understand your concern.
For more robustness, I'll add another catch-all clause that logs an exception.

Reply:
addressed with https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1143/commits/3f39974831857fc0e2abe7afc477ba17caa1c359